scout/server/blueprints/cases/templates/cases/case_sma.html

```diff
@@ -1,6 +1,7 @@
 {% extends "cases/case_tabular_view.html" %}
 {% from "utils.html" import comments_panel, activity_panel %}
 {% from "cases/utils.html" import individuals_table %}
```

Comment:
```suggestion
{% from "cases/utils.html" import individuals_table, sex_table_cell_content %}
```
Ah, perhaps include on row above! 😊

Reply:
Yes, makes sense!